proseco/characteristics_guide.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+
+CCD = {'row_min': -512.5,
+       'row_max': 511.5,
+       'col_min': -512.5,
+       'col_max': 511.5,
+       'window_pad': 7,
+       'row_pad': 8,
+       'col_pad': 1,
+       'bgpix': ['A1', 'B1', 'G1', 'H1', 'I4', 'J4', 'O4', 'P4']}
+
+PIX_2_ARC = 4.9628899999999998
```

Comment:
Did you decide about how to handle this on the `proseco.acq` side?

Reply:
I'm using 5 (5.0).  It shows up in two places converting a box size in arcsec to pixels.  For sizes less than 300" if you are rounding to an integer pixel then it gives the same answer.

For acq it basically does not matter since the probabilistic approach has fuzz built in, so being 0.1 pixels off in some margin calculation is not a problem.

Note that this level of precision is not actually justified since it changes by much more than that over the FOV.